internal/compiler/expression_tree.rs

```diff
@@ -39,6 +39,7 @@ pub enum BuiltinFunction {
     Pow,
     SetFocusItem,
     ShowPopupWindow,
+    ItemMemberFunction(String),
```

Comment:
Agreed.

Reply:
Agreed.

> > In the compiler this is still very primitive,
> 
> I think it's fine as is, no need to make it more complicated than it needs to be.
> 
> I wonder if there is really a use case for copy and paste.

I suppose for things like a "Copy" button (you know, the one you can sometimes see in websites with code snippets, the one you don't click to click on ;-)

> The alternative would be to have a `selected-text()` function (or out property?) that returns the selected text. Seems more usefull than copy/paste Perhaps also make the cursor and anchor pos public in-out properties.

Yes, a selected-text property would be great to have (in the future).

I'm very hesitant about exposing the cursor positions because it's unclear to me what the correct unit is. If they are utf-8 offsets it's a dangerous game.

I'm rather fond of the way we solved it in Qt with a dedicated cursor class and functions on it to navigate by logical means (next/prev word, end-of-line, etc.) - to avoid that the cursor is ever placed in an invalid position. Especially with rich-text later that becomes crucial.